mimic/rest/nova_api.py

```diff
@@ -219,8 +219,16 @@ def create_server(self, request, tenant_id):
             request.setResponseCode(400)
             return json.dumps(invalid_resource("Invalid JSON request body"))
 
-        return (self._region_collection_for_tenant(tenant_id)
-                .request_creation(request, content, self.url))
+        try:
+            creation = (self._region_collection_for_tenant(tenant_id)
+                        .request_creation(request, content, self.url))
+        except ValueError:
+            request.setResponseCode(400)
+            return json.dumps(
```

Comment:
In this case, the nova error when I tried it looks like it's:

```
{
    "badRequest": {
        "message": "OS-DCF:diskConfig must be either 'MANUAL' or 'AUTO'.",
        "code": 400
    }
}
```

Looks like the `invalid_resource` for the invalid JSON is also wrong.  Would you mind just changing the message to reflect the actual nova message, and I'll file another ticket to fix the use of invalid_resource for both this and the JSON message?